torchci/test/disableFlakyBot.test.ts

```diff
@@ -61,6 +74,27 @@ describe("Disable Flaky Test Bot Integration Tests", () => {
     }
   });
 
+  test("previously undetected flaky test should create an issue on main", async () => {
+    const scope = nock("https://raw.githubusercontent.com")
+        .get(`/pytorch/pytorch/master/test/${flakyTestB.file}.py`)
```

Comment:
Could you make an issue that after we do the switch we should also switch these to main?

Reply:
#299